x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_container.tsx

```diff
@@ -80,6 +81,19 @@ export const LatestFindingsContainer = ({ dataView }: FindingsBaseProps) => {
     [findingsGroupByNone.data?.total, urlQuery.pageIndex, urlQuery.pageSize]
   );
 
+  const handleDistributionClick = (evaluation: Evaluation) => {
```

Comment:
```suggestion
  const handleSetEvaluation = (evaluation: Evaluation) => {
```

IMO "distribution click" doesn't say much. `setEvaluation` says exactly what we're doing